src/__snapshots__/App.test.jsx.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@neelxie consider ignoring the **__snapshots__** in the **.gitignore**. I don't think we really need to keep track of these snapshot files.

Reply:
@neelxie snapshots should be verion controlled i.e. don't gitignore them